content/roadmaps/114-defi/content/111-yield-optimizers/109-popsicle.md

```diff
@@ -1 +1,7 @@
 # Popsicle
+
+Popsicle Finance is a multichain yield optimization platform for Liquidity Providers. It manages crypto investment by allocating it to various liquidity pools at DeXs to maximize yield. As such, it helps users take a more “hand-off” approach to yield optimization. It supports Ethereum, Avalanche, Fantom, BSC, Arbitrum and Polygon blockchains.
+
+<ResourceGroupTitle>Free Content</ResourceGroupTitle>
+<BadgeLink badgeText='Read' colorScheme='yellow' href='https://docs.popsicle.finance/'>Popsicle Docs</BadgeLink>
+<BadgeLink badgeText='Read' colorScheme='yellow' href='https://popsiclefinance.medium.com/welcome-to-popsicle-finance-9d0157f96921'>Welcome to Popsicle.Finance</BadgeLink>
```

Comment:
Seems official docs and intro article on medium are not the most fruity resource you could find 🤔 

Random article: https://blocmates.com/blogmates/a-complete-guide-to-popsicle-finance-ice-and-nice/ (again: probably not the best 😅 )